docs/00_getting_started_sd.md

```diff
@@ -1,40 +1,41 @@
-## First time setup of SD card
+## Initial Setup of SD Card
 
-### Flashcarts
-Using your PC, insert the SD card and ensure it is formatted for compatibility with your flashcart.
-**warning** Filenames are expected to be part of the ASCII character set. Unicode characters are not fully supported and may cause a crash screen.
```

Comment:
I think this is fixed now with the latest libdragon changes (at least, it works again with `720° ROM.txt`.

Reply:
Though I have not tried on a `fat32` formatted SD card.

Reply:
Are you sure? If it's solved, then my current rewrite should be rewritten to say that unsupported characters will display incorrectly (I haven't tried to add a Cyrillic or kanji-based name. How would those characters show up, with a question mark or...?).

Reply:
I've done a quick test with the latest rolling release. Right now Menu is compatible with ASCII and Latin-1 Extension characters, but not with Cyrillic. Trying to open a ROM with Cyrillic characters will not cause Menu to crash, it will just not show any characters. You might want to consider replacing characters that are not available in the font or compatible with Menu with question marks on the GUI.

This should clear the last review we had pending for this PR.